here-naksha-lib-core/src/test/resources/patcher/feature_3_patched_to_4_no_remove.json

```diff
@@ -0,0 +1,21 @@
+{
+  "foo": "bar",
+  "willBeUpdated": "updated",
+  "firstToBeDeleted": "not yet",
+  "isAdded": true,
+  "map": {
+    "mapID": "retained",
+    "willBeUpdated": "updated",
+    "willBeDeleted": "not yet",
+    "isAdded": true
+  },
+  "array": [
+    "retainedElement",
+    {
+      "id": "element1",
+      "willBeDeletedProperty": "not yet",
```

Comment:
I suggest also considering update of one property in array object

Reply:
I added an element for this

Reply:
I still don't see update field in array object... I am referring to adding "willBeUpdatedProperty" to array[2] object:
```
    {
      "id": "element1",
      "willBeUpdatedProperty": "some value",    ----> Adding this attribute
      "willBeDeletedProperty": "not yet"
    }
```